hazelcast/src/main/java/com/hazelcast/instance/DefaultAddressPicker.java

```diff
@@ -364,18 +365,16 @@ private void log(Level level, String message) {
     }
 
     private class InterfaceDefinition {
+
         String host;
         String address;
 
-        private InterfaceDefinition() {
```

Comment:
I removed the private constructor since CheckStyle insists on making a class final if it has just private constructors. But AddressDefinition extends InterfaceDefinition, so this won't work. Since the inner class is private, there is no need for private constructors anyway, so safe to delete.